platform/installation/offline-installations.mdx

```diff
@@ -21,6 +21,34 @@ To solve not being able ot pull images from Docker hub, you can
 [use a private image registry](#offline-images) to host your own copies of the vCluster Platform image that
 your cluster can reach.
 
+### Prerequisites
+
+- A running air-gapped Kubernetes cluster `v1.26.x`, `v1.27.x`, or `v1.28.x`
+- On installation computer:
+    - `kubectl` (check via `kubectl version`)
+    - `helm` (check with `helm version`)
+    - `docker` (check with `docker version`)
+    - A `kube-context` with admin access to this Kubernetes cluster (check with `kubectl auth can-i create clusterrole -A`)
+    - [Optional] An offline license key for vCluster.Pro (provided by Loft)
+- [Optional] A private docker registry that the installer computer and the air-gapped Kubernetes cluster can access
+
+### Resource requirements
+
+- vCluster.Pro manager pod resource requirements:
```

Comment:
Can we make this more generic to say `Pod Resource Requirements` in case we change the pod name later?

Reply:
yup yup, updated